components/ide/jetbrains/backend-plugin/launch-dev-server.sh

```diff
@@ -39,4 +39,7 @@ export IJ_HOST_SYSTEM_BASE_DIR=/workspace/.cache/JetBrains
 # Enable host status endpoint
 export CWM_HOST_STATUS_OVER_HTTP_TOKEN=gitpod
 
-$TEST_BACKEND_DIR/bin/remote-dev-server.sh run $TEST_DIR
\ No newline at end of file
+# Build and move ide-cli to its respective place.
+(cd ../cli && go build && mv cli /ide-desktop/bin/idea-cli)
```

Comment:
why not update PATH instead?

Reply:
💡 Good idea!
I'll update it.

I'll move it to `/ide-desktop/bin/idea-cli-dev` and then update the PATH, just for the file don't be listed as a newly-added-file-ready-to-commit to git.